fe/fe-core/src/test/java/com/starrocks/analysis/IndexDefTest.java

```diff
@@ -59,12 +64,31 @@ public void testAnalyzeExpection() {
         } catch (SemanticException e) {
             Assert.assertTrue(e instanceof SemanticException);
         }
+
+        def = new IndexDef("", null, IndexType.GIN, "");
+        Assertions.assertThrows(SemanticException.class, () -> def.analyze(), "Index can not accept null column.");
+
+        def = new IndexDef("", Lists.newArrayList("k1", "k2"), IndexType.BITMAP, "");
+        Assertions.assertThrows(SemanticException.class,
+                () -> def.analyze(), "bitmap index can only apply to a single column.");
+
+        def = new IndexDef("", Lists.newArrayList("k1", "k2"), IndexType.GIN, "");
+        Assertions.assertThrows(SemanticException.class,
+                () -> def.analyze(), "bitmap index can only apply to a single column.");
+
+
     }
 
     @Test
     public void toSql() {
         Assert.assertEquals("INDEX index1 (`col1`) USING BITMAP COMMENT 'balabala'", def.toSql());
         Assert.assertEquals("INDEX index1 ON table1 (`col1`) USING BITMAP COMMENT 'balabala'",
                 def.toSql("table1"));
+
+        Map<String, String> properties = new HashMap<>();
+        properties.put("k1", "k1");
+        def = new IndexDef("index1", Lists.newArrayList("k1", "k2"), IndexType.GIN, "", properties);
+        Assert.assertEquals("INDEX index1 ON table1 (`k1`,`k2`) USING GIN PROPERTIES(\"k1\"=\"k1\") COMMENT ''",
```

Comment:
```suggestion
        Assert.assertEquals("INDEX index1 ON table1 (`k1`,`k2`) USING GIN (\"k1\"=\"k1\") COMMENT ''",
```

Reply:
So toSql is not changed according to the new syntax.

Reply:
Yes, you are right